validator/testcases/regex.py

```diff
@@ -537,15 +537,15 @@ class Gecko53RegexTests(CompatRegexTestHelper):
     def tests(self):
         yield self.get_test_bug(
             1321556,
-            r'\burlbarBindings.xml\#splitmenu\b',
+            r'\burlbarBindings\.xml#splitmenu\b',
             'The splitmenu element has been removed.',
             'The splitmenu element has been removed.',
             log_function=self.err.warning,
             compat_type='error')
 
         yield self.get_test_bug(
             1331296,
-            r'\b[^-]*-moz-calc\b',
+            r'-moz-calc\b',
```

Comment:
Usually, if `-` is the last character in square brackets, it does not need to be escaped. I think the `*` made it slow, as the match can be quite large and thus take some time to compute.